functions/install/bedrock.sh

```
@@ -0,0 +1,73 @@
+# install_bedrock installs the Vanilla Bedrock Server: determines the appropriate download URL (latest or specific version), downloads and unpacks the archive, restores and updates configuration (including server port), optionally applies a forced MOTD, creates the service configuration, and launches the server.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)